cg/store/database.py

```diff
@@ -20,7 +20,7 @@ def initialize_database(db_uri: str) -> None:
     SESSION = scoped_session(session_factory)
 
 
-def get_session() -> scoped_session:
+def get_session() -> Session:
```

Comment:
It is a `scoped_session`, but it can be treated as a `Session` which is nice because then we get the proper syntax highlighting and auto completion as well 👍